postgresql-native/src/main/java/org/ballerinalang/postgresql/utils/Utils.java

```diff
@@ -177,4 +179,16 @@ private static void addSSLOptions(BMap secureSocket, BMap<BString, Object> optio
             }
         }
     }
+
+    public static Object validateResult(Object result) {
+        if (result instanceof BError &&  isContainString(result)) {
+            return ErrorGenerator.getSQLApplicationError(Constants.ERROR_MSG);
+        }
+        return result;
+    }
```

Comment:
Shall we not do additional checks and just pass the error we get from the SQL client. it is okay to have `SQL Client` in the error message